lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -585,6 +585,19 @@ def _determine_lims(self, xmin=None, xmax=None, *args, **kwargs):
             xmax += 0.05
         return (xmin, xmax)
 
+    def _validate_axis_limits(self, limit, convert):
+        """
+        If the axis limits being set are infinite, this function
```

Comment:
First line should be a full sentence; can be shortened to "Raise if specified axis limits are infinite."